test/parser.spec.ts

```diff
@@ -114,33 +114,18 @@ describe('gfm parser', () => {
 
     const expected = [
       notion.paragraph([notion.richText('hello')]),
-      notion.paragraph([
-        notion.richText('public class Foo {}', {
-          annotations: {code: true},
-        }),
-      ]),
+      notion.code([notion.richText('public class Foo {}')]),
     ];
 
     expect(actual).toStrictEqual(expected);
   });
 
   it('should parse block quote', () => {
-    const ast = md.root(
-      md.blockquote(
-        md.heading(1, md.text('hello '), md.emphasis(md.text('world')))
-      )
-    );
```

Comment:
Can this part of this test be added back? Or is it tested elsewhere?

Reply:
The reason it is removed is that to get block quotes to work they can only contain a single rich text, not an array of other blocks (as it can in Markdown):

https://developers.notion.com/reference/block#quote-blocks

So to allow this to work I decided that when parsing a block quote I only take the first paragraph and convert that to RichText - wasn't sure of a better option.

https://github.com/cliftonc/martian/blob/upgrade-fix/src/parser/internal.ts#L86

Reply:
The notion docs say quote blocks support an array of rich text objects. I think we would still want to keep the array in this test case and demonstrate how the emphasis on `world` becomes an annotation on the rich text object

Reply:
Ok - have addressed this in the latest commit, thanks for spotting!